components/ArtPieces/ArtPieces.js

```diff
@@ -0,0 +1,18 @@
+import ArtPiecePreview from "../ArtPiecePreview/ArtPiecePreview";
+
+export default function ArtPieces({ pieces }) {
+  return (
+    <>
+      {pieces.map((piece) => {
+        return (
+          <ArtPiecePreview
+            title={piece.name}
+            artist={piece.artist}
+            image={piece.imageSource}
+          ></ArtPiecePreview>
+        );
+      })}
+    </>
```

Comment:
I would like work with a list and use `ul` and `li` for your list of art pieces. :) And also the closing tag is not needed. And you should get an error in your console as you did not add a `key` prop or? 🤔